app/index.html

```diff
@@ -8,9 +8,11 @@
   <link rel="icon" type="image/png" href="../../images/openmrs-favicon.png" />
   <link rel="stylesheet" href="libs/bootstrap/dist/css/bootstrap.css" />
   <link rel="stylesheet" href="libs/toastr/toastr.min.css" />
+  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```

Comment:
Nice job. Installing these resource would mean that it can be accessible offline too as opposed to the dependency on internet connection for content delivery/availability.